packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -436,16 +444,32 @@ export class ConnectionManager implements IConnectionManager {
 
 		this._outbound.clear();
 
+		let text = "Closing DeltaManager";
+		let error: ICriticalContainerError | undefined;
```

Comment:
Doesn't look like it will ever be undefined. It is set in both if-else blocks below.